package.json

```diff
@@ -17,15 +17,17 @@
     "fix": "run-s fix:*",
     "fix:prettier": "prettier \"src/**/*.ts\" --write",
     "fix:lint": "eslint --fix . --ext .ts",
-    "ci:test": "run-s build test:lint test:unit test:integration",
+    "ci:test:staging": "run-s build test:lint test:unit test:integration:staging",
+    "ci:test:release": "run-s build test:integration:release",
     "ci:test-with-coverage": "nyc -r lcov -e .ts -x \"*.test.ts\" yarn ci:test",
     "docs": "./generate_docs",
     "generate:defs": "ts-node --skip-project node_modules/.bin/polkadot-types-from-defs --package @interlay/polkabtc/interfaces --input ./src/interfaces",
     "generate:meta": "ts-node --skip-project node_modules/.bin/polkadot-types-from-chain --package @interlay/polkabtc/interfaces --endpoint ./src/json/parachain.json --output ./src/interfaces",
     "test": "run-s build test:*",
     "test:lint": "eslint src --ext .ts",
     "test:unit": "mocha test/unit/*.test.ts test/unit/**/*.test.ts",
-    "test:integration": "mocha test/integration/**/*.test.ts --timeout 10000000",
```

Comment:
We could leave this to run both? Maybe you'd want to do that locally in a single command